lib/dpl/provider/transifex.rb

```diff
@@ -0,0 +1,46 @@
+module DPL
+  class Provider
+    class Transifex < Provider
+      experimental 'Transifex'
+
+      DEFAULT_CLIENT_VERSION = '>=0.11'
+      DEFAULT_HOSTNAME = 'https://www.transifex.com'
+
+      def install_deploy_dependencies
+        self.class.pip 'transifex', 'transifex', options[:cli_version] || DEFAULT_CLIENT_VERSION
```

Comment:
Can we put this `||` outside of the `#pip` call? The operator precedence is not clear.